src/commands/Deploy/test/appName.test.ts

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, test } from '@jest/globals';
+import { program } from 'commander';
+import { DIR_BUILD_PATH } from '../../../../unit_test/constant';
+import {
+  authCommandImplement,
+  createTemplate,
+  getRandomProjectName,
+  initProject
+} from '../../../../unit_test/helper';
+import deployCommand from '../deployCommand';
+
+const initTestProject = async () => {
+  const projectName = getRandomProjectName();
+
+  await initProject(DIR_BUILD_PATH, projectName);
+  await createTemplate(DIR_BUILD_PATH, projectName);
+  await authCommandImplement(program, process);
+};
+
+describe('Deploy command', () => {
+  describe('App name', () => {
+    test('Should be "test-app" when setting the value "test-app" to "--app-name" option', async () => {
```

Comment:
I think there is no need of the option description.
```suggestion
    test('Should be "test-app" when setting the value "test-app"', async () => {
```